website/content/es/_index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "CNCF TAG App Delivery"
```

Comment:
@lianmakesthings this is ok in English right? no need to translate it to Spanish, correct?